Cryptography/README.md

```diff
@@ -0,0 +1,9 @@
+### This objective of this project is to encode and decode messages using a common key. This project will be built using the Tkinter and base64 library.

```

Comment:
The readme file should have at least 2 -3 headings to describe different sections. My suggested sections: Title of the script, small description, any requirements, how to run, and at the last author name with your GitHub as a hyperlink (you should take credit for your work!)

You can check out other merged scripts readme for implementation or use this readme template as it is:
https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md

Reply:
ok @kaustubhgupta  I will improve that .